.github/ISSUE_TEMPLATE/Feedback_Currency_Dashboard.yml

```diff
@@ -0,0 +1,30 @@
+name: Feedback for Currency Dashboard
+title: "[Feedback]:"
+description: "Feedback for currency dashboard"
+labels: "dashboard feedback"
+projects: "ppc64le/1"
+assignees: "ICH"
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Please fill in the following details to provide feedback or your suggstion for Currency Dashboard.
+  - type: dropdown
+    id: dashboard_version
+    attributes:
+      label: Currency Dashboard Version
+      options:
+        - v0.5
+        - v0.6
+        - v1.0
```

Comment:
Again, can someone actually report a defect on a previous (or future?) version of the portal?  But we definitely want to save the version of the portal they reported against?

Reply:
covered it